deepsea/index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Deep Sea Theme</title>
+    <link rel="icon" href="assets/favicon.ico" type="image/ico" sizes="16x16" />
+    <meta name="description" content="Deep Sea theme create by Rane G" />
+    <meta name="theme-color" content="#134074" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel='apple-touch-icon' href='/logo.png'>
+    <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+    <header>
+        <nav id="nav-bar" class="nav-bar">
+            <div class="nav-logo">
+                <img class="logo" alt="Deep Sea Logo" src="assets/logo.png" />
+                <a href="#hero">
+                    <span>Deep Sea</span>
+                </a>
+            </div>
+
+            <div class="nav-links">
+                <ul>
+                    <li>
+                        <a class="nav-links-item" href="#">Home</a>
+                    </li>
+                    <li>
+                        <a class="nav-links-item" href="#about">About</a>
+                    </li>
+                    <li>
+                        <a class="nav-links-item" href="#portfolio">Portfolio</a>
+                    </li>
+                    <li>
+                        <a class="nav-links-item" href="#contact">Contact</a>
+                    </li>
+                </ul>
+            </div>
+
+            <div class="nav-menu">
+                <img id="nav-icon" src="assets/hamburger.svg" alt="Menu Icon" class="hamburger-icon" />
+            </div>
+        </nav>
+    </header>
+
+    <main>
+        <section id="hero">
+            <div class="hero-text">
+                <p class="greet-text">Hi there! I am</p>
+                <p class="text-gradient name-text">Rane G.</p>
+                <p class="job-text">Web Developer.</p>
+                <p class="intro-text">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Suscipit, obcaecati!
+                    Soluta nemo consequatur, natus facilis, suscipit recusandae pariatur consequuntur ex accusantium
+                    repudiandae incidunt placeat debitis eius fugiat quod eos nam!
+                </p>
+                <a href="#contact" class="contact-button">Contact Me</a>
+            </div>
+            <div class="hero-image">
+                <img class="hero-image" src="assets/hero.png" width="512" alt="Hero Image" />
+            </div>
+        </section>
+
+        <section id="about">
+            <h2 class="text-gradient">About</h2>
+            <div>
+                <div>
+                    <img src="assets/logo.png" width="128" height="128" alt="Person Image" />
+                </div>
+                <ul>
+                    <li>
+                        <img class="about-icon" src="assets/user.svg" alt="person icon" />
+                        <p>Rane Gillian Villanueva</p>
```

Comment:
We need to make the templates to have as less promotion as possible so if we could replace personal pics and names with placeholders, that would be better.

Reply:
In short, no self promotion, I hope you understand. Thanks.

Reply:
Yeah. No problem.